packages/Pagination/hash.js

```diff
@@ -0,0 +1,26 @@
+import stringHash from 'string-hash'
```

Comment:
Why do we need this hash utility? Can we use the index as the key?

Reply:
index as key won't work as its not always unique